cfme/tests/control/test_bugs.py

```diff
@@ -147,6 +150,51 @@ def hardware_reconfigured_alert(alert_collection):
     alert.delete()
 
 
+@pytest.fixture
+def setup_disk_usage_alert(appliance):
+    # get the current time
+    timestamp = datetime.now()
+    # setup the DB query
+    table = appliance.db.client['miq_alert_statuses']
+    query = appliance.db.client.session.query(table.description, table.evaluated_on)
+    # configure the advanced settings and place a large file on the appliance
+    # disk usage above 1 % will now trigger a disk_usage event
+    appliance.update_advanced_settings(
+        {"server": {"events": {"disk_usage_gt_percent": 1}}}
+    )
+    # create a 1 GB file on /var/www/miq/vmdb/log
+
+    result = appliance.ssh_client.run_command(
+        "dd if=/dev/zero of=/var/www/miq/vmdb/log/delete_me.txt count=1024 bs=1048576"
```

Comment:
OPTIONAL: you'd better use 1M here for readability.
and one more suggestion. I'd consider using truncate instead of dd. it should be quicker and more readable.